packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts

```diff
@@ -2,13 +2,14 @@ import { IButtonStyles } from '../Button.types';
 import { ITheme, concatStyleSets, getFocusStyle, HighContrastSelector } from '../../../Styling';
 import { memoizeFunction } from '../../../Utilities';
 import { getStyles as getBaseButtonStyles } from '../BaseButton.styles';
-import { getStyles as getSplitButtonStyles } from '../SplitButton/SplitButton.styles';
+import { getStandardSplitStyles } from '../SplitButton/SplitButton.styles';
```

Comment:
Can we call it `getStandardSplitButtonStyles` instead?

Ditto with the primary version.